packages/error-debugging/dev-toolbar/src/apps/settings/settings-app.tsx

```
@@ -0,0 +1,332 @@
+            <div class="text-[0.8125rem] font-medium text-foreground leading-none mb-0.5">{label}</div>
+            {description && <div class="text-[0.725rem] text-muted-foreground leading-snug mt-1">{description}</div>}
+        </div>
+        <div class="shrink-0">{control}</div>
+    </div>
+);
+
+/** Section with title and divider */
+const Section = ({ title, children }: { title: string; children: ComponentChildren }): ComponentChildren => (
+    <section>
+        <h3 class="text-[0.65rem] font-bold uppercase tracking-[0.1em] text-muted-foreground mb-2 px-1 flex items-center gap-1.5">
+        <span aria-hidden="true" class="text-primary/50">//</span>
```

Biome (2.4.4)

[error] Wrap comments inside children within braces.

(lint/suspicious/noCommentText)